external/dhcpc/dhcpc.c

```diff
@@ -673,45 +691,37 @@ static int dhcpc_request(void *handle, struct dhcpc_state *presult)
 #if defined CONFIG_NET_LWIP    // this is temporal fix. it should be modified later
     ip_addr_t dns_addr;
```

Comment:
you have to use tab instead of space.

Reply:
I did no change in this line so i didn't check it.  now corrected